org.freedesktop.Sdk.Extension.golang.json

```diff
@@ -0,0 +1,87 @@
+{
+  "id": "org.freedesktop.Sdk.Extension.golang",
+  "branch": "1.6",
+  "runtime": "org.freedesktop.Sdk",
+  "build-extension": true,
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk-extensions": [],
+  "separate-locales": false,
+  "appstream-compose": false,
+  "modules": [
+    {
+      "name": "golang",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": [
+            "i386"
+          ],
+          "url": "https://dl.google.com/go/go1.9.2.linux-386.tar.gz",
+          "sha256": "574b2c4b1a248e58ef7d1f825beda15429610a2316d9cbd3096d8d3fa8c0bc1a"
+        },
+        {
+          "type": "archive",
+          "only-arches": [
+            "arm"
+          ],
+          "url": "https://dl.google.com/go/go1.9.2.linux-armv6l.tar.gz",
```

Comment:
I'm pretty sure this is the wrong arm architecture for our definition of ARM. We should test, but we may have to ban arm from this extension

Reply:
Ah, right, we're targetting armv7? I could probably make some workaround to build it from source, I'll take a look at it.

Reply:
If we can just build it from source that would be nice in general.

Reply:
After another look, seems to be potential time sink. Last release that doesn't require working Go compiler is 1.4 and it doesn't seem to build on my Odroid C2 (aarch64).

One option would be another extension to build this one (that would also cut some time for building), but use binary release just for aarch64.

Reply:
Yeah - I'd rather have go, and the bootstrap is super messy.

Reply:
@ramcq which exact arm do we use again? you were the last person who touched that as I remember ;-)

Reply:
Looking at how gcc is built, it's armv7-a. Fix will come in a minute…

Reply:
My bad, there are no binaries optimized for armv7. I will test armv6 one on my Odroid XU4 and merge it as is if it works properly.